examples/shell/shell_common/cmd_ping.cpp

```diff
@@ -367,8 +367,9 @@ void PrintUsage(streamer_t * stream)
                     "  -t              use TCP\n"
                     "  -p  <port>      echo server port\n"
                     "  -i  <interval>  ping interval time in seconds\n"
-                    "  -c  <count>     stop after <count> replies\n"
-                    "  -r  <1|0>       enalbe/disable CRMP\n");
+                    "  -c  <count>     stop after <count> replies\n");
+
+    streamer_printf(stream, "  -r  <1|0>       enable or disable CRMP\n");
```

Comment:
This split location is not obvious.

How about splitting every output individually (to make it consistent) and adding a comment of why things are split.
I would not expect streamer_printf to have limits. Why does it have one?

Reply:
Acked,  the size of streamer_printf buffer is set to 256.
#define CONSOLE_DEFAULT_MAX_LINE 256